_custom_build/backend.py

```diff
@@ -54,5 +27,15 @@ def build_wheel(
         self.config_settings = config_settings
         return super().build_wheel(wheel_directory, config_settings, metadata_directory)
 
+    def build_editable(
+        self, wheel_directory, config_settings=None, metadata_directory=None
+    ):
+        self.config_settings = config_settings
+        return super().build_editable(
+            wheel_directory, config_settings, metadata_directory
+        )
+
 
-build_wheel = _CustomBuildMetaBackend().build_wheel
+_backend = _CustomBuildMetaBackend()
+build_wheel = _backend.build_wheel
+build_editable = _backend.build_editable
```

Comment:
```suggestion
if not getattr(_backend, "LEGACY_EDITABLE", False):
    build_editable = _backend.build_editable
```
Looking at https://github.com/pypa/setuptools/blob/6ee23bf0579c52e1cbe7c97fc20fd085ff2a25c7/setuptools/build_meta.py#L430-L437, I think perhaps this condition should be added, because there may not be a `super()` method to call.

I've used `getattr` to guard against it being [removed in the future.](https://github.com/pypa/setuptools/blob/6ee23bf0579c52e1cbe7c97fc20fd085ff2a25c7/NEWS.rst#L844-L847)

Reply:
I'm not sure it is even possible to use LEGACY_EDITABLE mode anymore, since all my attempts end with 
```
    File "C:\Git\Pillow\_custom_build\backend.py", line 5, in <module>
      from setuptools.build_meta import *  # noqa: F403
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  AttributeError: module 'setuptools.build_meta' has no attribute 'get_requires_for_build_editable'
```
It seems pip will not even attempt to use `build_editable` without first seeing `get_requires_for_build_editable` which is not defined here, so I'm not sure this change is necessary?

Reply:
Actually, this is failing on `from setuptools.build_meta import *` due to `get_requires_for_build_editable` being listed in `setuptools.build_meta.__all__`. After changing it to only import the relevant functions, I can finally build an editable wheel, but it is ignoring all `--config-settings`. So I'm not sure there is a need to support legacy mode when it is already non-functional and there have been no complaints AFAIK.

Reply:
Ok. Thanks for investigating.